src/components/ActivityModal.tsx

```
@@ -0,0 +1,201 @@
+                  <h2 className="text-2xl font-bold text-camp-text-modalTitle mb-2">
+                    {data.section.name}
+                  </h2>
+                  <p className="text-lg text-camp-text-secondary">
+                    {formatDateRange(data.section.startDate, data.section.endDate)}
+                  </p>
+                </div>
+
+                {/* Second Row: Block and Activity */}
+                <div className="bg-camp-background rounded-lg p-6 border border-gray-200">
+                  <h3 className="text-xl font-bold text-camp-text-modalTitle">
+                    BLOCK {blockId.toUpperCase()} - "{activity.name.toUpperCase()}"
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)